src/main/java/org/jvnet/hudson/plugins/repositoryconnector/Artifact.java

```diff
@@ -49,6 +49,11 @@ public String getArtifactId() {
      * @return the version
      */
     public String getVersion() {
+    	if(version==null) {
+    		return null;
+    	} else if (version.trim().length()==0) {
+    		return null;
+    	}
```

Comment:
yes. i'm not so familiar with the available utils. do you want me to change this?